lerobot/common/datasets/factory.py

```diff
@@ -81,8 +81,26 @@ def make_dataset(cfg: TrainPipelineConfig) -> LeRobotDataset | MultiLeRobotDatas
     image_transforms = (
         ImageTransforms(cfg.dataset.image_transforms) if cfg.dataset.image_transforms.enable else None
     )
-
-    if isinstance(cfg.dataset.repo_id, str):
+    if cfg.dataset.repo_id.startswith('['):
```

Comment:
in omegaconfig this used to be a ListConfig, now it's a string with square brackets so we can just parse it.